tests/conftest.py

```diff
@@ -251,14 +251,16 @@ def _get_table(
     instance_key: Optional[str] = None,
 ) -> AnnData:
     if region is not None:
-        region_key = region_key or "annotated_region"
         instance_key = instance_key or "instance_id"
     adata = AnnData(RNG.normal(size=(100, 10)), obs=pd.DataFrame(RNG.normal(size=(100, 3)), columns=["a", "b", "c"]))
     if instance_key is not None:
+        # from today pycharm refuses to execute this line if I run a test...
```

Comment:
unrelated but there are a bunch of lines of code or comments with # pycharm can or cannot do something around the repo. I would consider to remove them cause they clutter the code and introduce lines/asserts 😅 I would put code quality and consistency with higher priority than whether pycharm works or not in any specific case

Reply:
yeah I agree; I'll stop using the `mypy` extensions (it's anyway super slow), which is the cause of most of the comments. This one above is actually false, I'll remove it.